org.omg.sysml/src/org/omg/sysml/expressions/functions/MetaFunction.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * SysML 2 Pilot Implementation
+ * Copyright (c) 2022 Model Driven Solutions, Inc.
+ *    
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *  
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  
+ * @license LGPL-3.0-or-later <http://spdx.org/licenses/LGPL-3.0-or-later>
+ *  
+ *******************************************************************************/
+
+package org.omg.sysml.expressions.functions;
+
+public class MetaFunction extends AsFunction {
+
+	@Override
+	public String getOperatorName() {
+		return "meta";
+	}
+
```

Comment:
Where is the code that does the ".metadata" now? This is now a simple casting only.

Reply:
`.`**`metadata`** is not actually an operator. It corresponds to a MetaclassAccessExpression in the abstract syntax. Therefore, it is handled directly by the new `ModelLevelExpressionEvaluator::evaluateMetadataAccess` method (added in commit [7f40d80](https://github.com/Systems-Modeling/SysML-v2-Pilot-Implementation/pull/416/commits/7f40d809f8917abe5a415c9adddb2134c5ef64f1)).

An expression like `e`**`meta`**`T` is actually parsed with the first argument being `e.`**`metadata`**, so all the `'meta'` function needs to do is the cast (i.e., it is the same as `'@'`).